src/python/detectors/notebook_invalid_execution_order/notebook_invalid_execution_order_noncompliant.ipynb

```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "####  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\n",
+    "####  SPDX-License-Identifier: Apache-2.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# {fact rule=notebook-invalid-execution-order@v1.0 defects=0}\n",
+    "# Noncompliant: execution order of cells is invalid.\n",
```

Comment:
execution order of cells is not linear. When executing cells in linear order, variables `x` and `y` are undefined before use.